eservices-monitoring-exporter/src/main/resources/application-standalone.conf

```diff
@@ -1,16 +1,15 @@
-interop-be-eservices-monitoring-exporter {
```

Comment:
Doesn't this change require a corresponding modification in the config reading section?

Reply:
It works perfectly